platform/ui/src/components/StudyListTable/StudyListTableRow.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const StudyListTableRow = (props) => {
+  const { tableData } = props;
+  const { row, expandedContent, onClickRow, isExpanded } = tableData;
+
+  const tdClasses = [
```

Comment:
We have 3 different td and only one is using this classes, maybe we should just add this in the jsx as the other ones.